com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+                        if ((tmp = EditorGUI.Slider(rect, Styles.fallOff, fallOffProp.floatValue, 0.0f, 1.0f)) != fallOffProp.floatValue)
+                            fallOffProp.floatValue = Mathf.Max(tmp, 0.0f);
+
+                        if (newType == SRPLensFlareType.Polygon)
+                        {
+                            rect = GetNextRect();
+                            if ((tmp = EditorGUI.IntSlider(rect, Styles.sideCount, sideCountProp.intValue, 3, 32)) != sideCountProp.intValue)
+                                sideCountProp.intValue = (int)Mathf.Max(tmp, 0);
+
+                            rect = GetNextRect();
+                            if ((tmp = EditorGUI.Slider(rect, Styles.sdfRoundness, sdfRoundnessProp.floatValue, 0.0f, 1.0f)) != sdfRoundnessProp.floatValue)
+                                sdfRoundnessProp.floatValue = Mathf.Clamp01(tmp);
+                        }
+
+                        rect = GetNextRect();
+                        if ((tmpBool = EditorGUI.Toggle(rect, Styles.inverseSDF, inverseSDFProp.boolValue)) != inverseSDFProp.boolValue)
+                            inverseSDFProp.boolValue = tmpBool;
+                    }
+
+                    rect = GetNextRect();
+                    if ((tmpBool = EditorGUI.Toggle(rect, Styles.preserveAspectRatio, preserveAspectRatioProp.boolValue)) != preserveAspectRatioProp.boolValue)
+                        preserveAspectRatioProp.boolValue = tmpBool;
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Color", EditorStyles.boldLabel);
+                {
+                    rect = GetNextRect();
+                    if ((tmpCol = EditorGUI.ColorField(rect, Styles.tint, tintProp.colorValue)) != tintProp.colorValue)
+                        tintProp.colorValue = tmpCol;
+                    rect = GetNextRect();
+                    if ((tmpBool = EditorGUI.Toggle(rect, Styles.modulateByLightColor, modulateByLightColor.boolValue)) != modulateByLightColor.boolValue)
+                        modulateByLightColor.boolValue = tmpBool;
+                    rect = GetNextRect();
+                    if ((tmp = EditorGUI.FloatField(rect, Styles.intensity, intensityProp.floatValue)) != intensityProp.floatValue)
+                        intensityProp.floatValue = Mathf.Max(tmp, 0.0f);
+                    rect = GetNextRect();
+                    SRPLensFlareBlendMode newBlendMode;
+                    SRPLensFlareBlendMode blendModeValue = (UnityEngine.SRPLensFlareBlendMode)blendModeProp.enumValueIndex;
+                    if ((newBlendMode = ((SRPLensFlareBlendMode)(EditorGUI.EnumPopup(rect, Styles.blendMode, blendModeValue)))) != blendModeValue)
+                        blendModeProp.enumValueIndex = (int)newBlendMode;
+                }
+
+                rect = GetNextRect();
+                EditorGUI.TextArea(rect, "Transform", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either **L18n** or **EditorGUIUtility.TrTextContent**